### DISCUSSION TITLE
interlace effect for beta0006

### DISCUSSION THREAD

**Discussion by jimcahum** [SixLabors/ImageSharp#1248] [Q&A] [answered]
Hello, I was using SixLabor rc-00001 version. My project framework need to keep 4.7.1 for some reason. It is working well locally. But got following error while deploying on Jenkins:

> Could not load file or assembly 'System.Numerics.Vectors, Version=4.1.3.0. Even I have updated System.Numerics.Vectors = 4.1.4.0. 

I guess the issue caused by default version of framework 4.7.1 is 4.1.3.0.
Therefore I chose previous version beta0006 which was not requiring System.Numerics.Vectors 4.1.4.0. Do you know how to add interlace effect for this version?InterlaceMethod = PngInterlaceMode.Adam7
Here is my code. Thanks!
```
var encoder = new PngEncoder
{
ColorType = PngColorType.Palette,
FilterMethod = SixLabors.ImageSharp.Formats.PngFilterMethod.None,
CompressionLevel = 1,
BitDepth = PngBitDepth.Bit8,
Quantizer = new WuQuantizer(16)
};
```

**Comment by JimBobSquarePants** [comment, marked as answer]
Yeah you've got a framework redirect issue. See this post for workarounds. 

https://nickcraver.com/blog/2020/02/11/binding-redirects/

Downgrading to beta6 is an unsupported solution. We cannot and never will support you doing that.

**Reply by jimcahum** [reply]
Thanks for your sharing. Do you think framework 4.7.1 could support any version from rc-00001 to rc-00003? Because I don't know why my  System.Numerics.Vectors always point to 4.1.3.0 even I have updated nugets package and bindingredirect.

**Reply by JimBobSquarePants** [reply]
It's not an ImageSharp issue. The cause and fix is mentioned in the article. You need to configure binding redirects.
